frontend/app/src/components/home/MakeProposalModal.svelte

```diff
@@ -63,16 +62,17 @@
     $: cryptoBalance = $cryptoBalanceStore[ledger] ?? BigInt(0);
     $: symbol = tokenDetails.symbol;
     $: howToBuyUrl = tokenDetails.howToBuyUrl;
-    $: transferFees = tokenDetails.transferFee;
-    $: requiredFunds = proposalCost + transferFees + transferFees;
+    $: transferFee = tokenDetails.transferFee;
+    $: requiredFunds = proposalCost + transferFee + transferFee;
     $: insufficientFunds = cryptoBalance < requiredFunds;
     $: padding = $mobileWidth ? 16 : 24; // yes this is horrible
     $: left = step * (actualWidth - padding);
     $: token = treasury === "SNS" ? symbol : "ICP";
     $: titleValid = title.length >= MIN_TITLE_LENGTH && title.length <= MAX_TITLE_LENGTH;
     $: urlValid = url.length <= MAX_URL_LENGTH;
     $: summaryValid = summary.length >= MIN_SUMMARY_LENGTH && summary.length <= MAX_SUMMARY_LENGTH;
-    $: amountValid = isAmountValid(amount);
+    $: amountE8s = Number(amount) * Number(Math.pow(10, tokenDetails.decimals));
```

Comment:
Shouldn't really be called e8s because there is no guarantee that these tokens have 8 decimals, for example DKP is about to switch to 3 decimals.

Reply:
I think just call it `amount`